### PR TITLE
Add seqcomplexity

### PR DESCRIPTION
**Pull request by stevenweaver**
Package that calculates per-read and total sequence complexity from FastQ file.

https://crates.io/crates/seqcomplexity

### CODE REVIEW

File: recipes/seqcomplexity/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - expat
+  run:
+
+test:
+  commands:
+    - seqcomplexity --help
+
+about:
+  home: https://github.com/stevenweaver/seqcomplexity
+  license: MIT
```
Comment (bgruening) [inline recipes/seqcomplexity/meta.yaml:39]: please add license_file: LICENSE

File: recipes/seqcomplexity/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+source:
+  url: https://github.com/stevenweaver/seqcomplexity/archive/v{{ version }}.tar.gz
+  sha256: 18db9c4e81e859c366c208e7c9b6286bd221b9ac9ebc238917ad36fdb68d292c
+
+requirements:
+  build:
+    - rust >=1.50
+    - clangdev
+    - cmake
```
Comment (bgruening) [inline recipes/seqcomplexity/meta.yaml:19]: cmake and make should be in the host section

**Comment by stevenweaver** [comment]
All checks have passed. make and cmake in the host section causes the run to fail and is not in similar packages (See fastqc-rs). 

Best,
Steven

**Comment by stevenweaver** [comment]
@bgruening -- bump

**Review by bgruening — APPROVED** [review]

**Pull request merged**